src/System.Runtime.WindowsRuntime/src/System/InternalHelpers.cs

```diff
@@ -43,38 +43,6 @@ internal static void TryDeregister(this CancellationTokenRegistration ctr)
 
     internal class Helpers
     {
-        private static Task s_completedTask = null;
-
-        internal static Task CompletedTask
-        {
-            get
-            {
-                var completedTask = s_completedTask;
-                if (completedTask == null)
-                {
-                    var taskSource = new TaskCompletionSource<VoidValueTypeParameter>();
-                    taskSource.SetResult(default(VoidValueTypeParameter));
-                    s_completedTask = completedTask = taskSource.Task;
-                }
-                return completedTask;
-            }
-        }
-
-        internal static Task<TYPE> TaskFromException<TYPE>(Exception e)
-        {
-            var taskSource = new TaskCompletionSource<TYPE>();
-            taskSource.SetException(e);
-            return taskSource.Task;
-        }
-
-        internal static Task<TYPE> TaskFromCancellation<TYPE>(CancellationToken cancellationToken)
-        {
-            var taskSource = new TaskCompletionSource<TYPE>();
-            taskSource.SetCanceled();
-            taskSource.Task.Wait(cancellationToken);
-            return taskSource.Task;
-        }
-
         internal unsafe static void ZeroMemory(byte* src, long len)
         {
             while (len-- > 0)
```

Comment:
Potentially, but a) it's unrelated to this PR and I'm not planning to change it here, b) this helper is used only in an internal copy of UnmanagedMemoryStream in System.Runtime.WindowsRuntime and only in some relatively rare circumstances (e.g. when writing beyond the current end of the stream and needing to zero out memory between the current end and the write), and c) I'd hope that this private copy could be removed entirely in favor of just using System.IO.UnmanagedMemoryStream.dll in corefx... so updating this helper to be more clever doesn't seem particularly high priority.  You're welcome to open an issue to track it, though, if you feel differently.